command/sub8_missions/sub8/sub_singleton.py

```diff
@@ -312,6 +312,17 @@ def _run_look(self, spotings_req):
 
             yield self.sub.nh.sleep(.5)
 
+class PoseSequenceCommander(object):
```

Comment:
I think that's the use case we're concerned with, 2 or 3. It would be great to have.